src/dotnet/Fable.JS/demo/rollup.config.js

```diff
@@ -5,14 +5,15 @@ function resolve(filePath) {
   return path.resolve(__dirname, filePath)
 }
 
-// var babelOptions = {
-//   "presets": [
-//     ["es2015", {"modules": false}]
-//   ]
-// };
+var babelOptions = {
+  "presets": [
+    // ["es2015", { "modules": false }],
+    // ["minify", { "mangle": false }],
+  ]
+};
```

Comment:
Is the Fable plugin causing problems if you don't pass Babel Options with an empty `presets` array? If that's the case, I guess we need to fix it?

Reply:
@alfonsogarciacaro I forgot to comment the babel options, but no, it doesn't seem to cause problems.